client/app/models/Vcf.iobio.js

```diff
@@ -1467,7 +1467,7 @@ export default function vcfiobio(theGlobalApp) {
   }
 
   exports._parseHeaderForInfoORFormat = function(record) {
-    const matches = record.match(/<ID=(\w+),Number=(\w+),Type=(\w+),Description="([^"]+)">/);
+    const matches = record.match(/<ID=(\w+),Number=(\w+|\.),Type=(\w+),Description="([^"]+)">/);
     if (matches) {
```

Comment:
It would be good to report any bypassed format fields due to a mismatch on the reg expression. I'm going to accept and merge this pull request and then add some error reporting in an else statement.